.docker/parameters.yml

```diff
@@ -29,5 +29,7 @@ parameters:
     xpub_client_id: some-id
     xpub_client_secret: ThisIsASecret
     subject_rewrites: []
+    rds_articles:
+        16846: https://repro.elifesciences.org/example.html
```

Comment:
This will work as long as we can provide the same reference for all versions of an article.

Reply:
I've checked locally and if we introduce leading zeros then the id will not match. My recommendation is that we declare the id as a string:

```
rds_articles:
    '16846': https://repro.elifesciences.org/example.html
```

Reply:
Can you confirm that we will only support one rds_article for a single paper?

Reply:
As far as I know yes, there is only one single example of a single paper, which probably points to one specific version (definitely not the PoA): https://repro.elifesciences.org/example.html

Checking the leading zeros: not sure what come out of the API for us to match.